### PR TITLE
feat: introduction - german [INT2-92]

### PR DESCRIPTION
**Pull request by Nesyeth**
Hi Phibert, hope i got it it right. :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
 
-The Yearn ecosystem is controlled by YFI token holders who submit and vote on off-chain proposals that govern the ecosystem. Proposals that generate a majority support \(&gt;50% of the vote\) are implemented by a 9 member multi-signature wallet. Changes must be signed by 6 out of the 9 wallet signers in order to be implemented. The members of the multi-signature wallet were voted in by YFI holders and are subject to change from future governance votes. Please refer to our FAQ for [the list of the multisig signers](https://docs.yearn.finance/resources/faq#who-is-on-the-multisig) and more information about the [governance process](https://docs.yearn.finance/resources/faq#governance).
-
-## Communication Channels
+## Kommunikationskanäle
 
 Governance Forum [https://gov.yearn.finance/](https://gov.yearn.finance/)
```
Comment (philburrrt) [inline README.md:22]: @Nesyeth yep you did it perfectly! looks like you missed this last line though

**Comment by Nesyeth** [comment]
Hello Phibert, not sure if i get what i need to change on the last line. Any input? and sorry if I am asking too many useless questions :)

**Comment by philburrrt** [comment]
> Hello Phibert, not sure if i get what i need to change on the last line. Any input? and sorry if I am asking too many useless questions :)

Oh, it just looked like 'Governance Forum' wasn't translated. Do you think it should be left as english?

**Review by Nesyeth — COMMENTED** [review]
@philburrrt There are a few english words that have similar meaning in german and any German reader will easily understand that . Governance has another word in german but will not fit the intended meaning.

**Comment by philburrrt** [comment]
#GERMAN INTRO

**Pull request merged**